js/containers/bottom-controls.js

```diff
@@ -165,6 +222,21 @@ class BottomControls extends PureComponent {
     log('ResetClicked')
   }
 
+  handleRecentToggle () {
+    const { filters, setFilter } = this.props
+    const currentDate = Date.now()
+    const monthAgo = currentDate - thirtyDays
+    // toggle between original start/end and the "recent 30 day" view
+    const startDate = filters.get('minTimeLimit') !== filters.get('initialStartTime') ? filters.get('initialStartTime') : monthAgo
+    const endDate = filters.get('maxTimeLimit') !== filters.get('initialEndTime') ? filters.get('initialEndTime') : currentDate
+
+    setFilter('minTime', startDate)
+    setFilter('maxTime', endDate)
+
+    setFilter('minTimeLimit', startDate)
+    setFilter('maxTimeLimit', endDate)
+  }
```

Comment:
I think some issues might be caused by the fact that this function doesn't set `playbackMaxTime`.

Reply:
Thanks! Yes, setting that filter seems to have fixed it.